test/main.test.js

```diff
@@ -6,7 +9,10 @@ import _init, { init } from "../module.js";
 
 describe("sqlparser", function () {
   before(async function () {
-    await init();
+    const _dirname = dirname(fileURLToPath(import.meta.url))
+    const file = fs.readFileSync(resolve(_dirname, "..", "main.wasm"));
+    const buffer = new Uint8Array(file).buffer;
+    await init(buffer);
```

Comment:
Since the tests are running in node without a server to serve the wasm, I think we have to provide the wasm in memory.

Reply:
This is another good reason to also embed the wasm in the browser/esm build as well as the cjs build.

Reply:
(eventually)